lib/bitcoin/BitcoinClient.ts

```diff
@@ -387,9 +398,16 @@ export default class BitcoinClient {
     };
 
     const response = await this.rpcCall(request, true);
+
+    if (!response.feerate ||
+        (response.errors && response.errors.length > 0)) {
+      const error = response.errors ? JSON.stringify(response.errors) : `Feerate is undefined`;
+      throw new Error(`Fee rate could not be estimated. Error: ${error}`);

```

Comment:
Sanity: Should we throw SidetreeError?

Reply:
The BitcoinClient does not throw any SidetreeErrors so I want to change it all together. Let me open an issue for that.